include/experimental/__p0009_bits/layout_stride.hpp

```diff
@@ -202,9 +202,17 @@ struct layout_stride {
         return __strides_storage_t{static_cast<index_type>(s[Idxs])...};
       }
 
-      template<class IntegralType>
+      MDSPAN_TEMPLATE_REQUIRES(
+        class IntegralType,
+        // The is_convertible condition is added to make sfinae valid
```

Comment:
It looks like this is rebased atop PR #336.  Should PR #336 be merged first?

Reply:
Yeah already done.